backend/app/Http/Controllers/EmployeeController.php

```diff
@@ -140,9 +140,30 @@ public function updateByID(UpdateEmployeeRequest $request, string $employeeId):
     public function confirmCSV(Request $request)
     {
         $parser = new CsvParser();
+        $passed = 0; $failed = 0;
         $file = json_decode($request->getContent(), true);
-        $data = $parser->trimEmployeeCSV($file['data'], $file['department_id'], $file['org_id']);
-        return $this->insertEmployee($data);
+        $json = $file['data'];
+        foreach($json as $key => $item){
+            // check if employe exists
+            $empExists = Employee::where("email", $json[$key]["email"]);
+
+            if ($empExists->count() > 0) {
+                return $this->sendResponse(true, "employee already exists", "employee with this email already exists", null, Response::HTTP_BAD_REQUEST);
+            }
+
+            $raw_password = implode("", $this->generateRandomPwd(10));
```

Comment:
why are you imploding here?

Reply:
its for generating the temporary password

Reply:
I  couldn't add screenshots because I'm getting a "Could not send request" response